demo/src/app/workbasket-filters-consumer.component.ts

```diff
@@ -0,0 +1,120 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
The only extracted component we expose should be the new case list filters. I think we should remove workbasket filters from the demo app as discussed before.

Reply:
Done